src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -853,7 +853,7 @@ class PCReferenceProcessor: public ReferenceProcessor {
       ReferenceProcessor(is_subject_to_discovery,
       ParallelGCThreads,   // mt processing degree
       ParallelGCThreads,   // mt discovery degree
-      true,                // atomic_discovery
+      false,               // concurrent_discovery
```

Comment:
It would be nice to improve the indentation of this code, i.e. the parameters of the `ReferenceProcessor` super class constructor call should be indented.